test/e2e/operator_test.go

```diff
@@ -4323,6 +4405,29 @@ func assertServiceAnnotation(t *testing.T, serviceName types.NamespacedName, ann
 	}
 }
 
+// assertLoadBalancerServiceAnnotationWithPollImmediate checks if the specified annotation on the
+// LoadBalancer Service of the given IngressController matches the expected value.
+func assertLoadBalancerServiceAnnotationWithPollImmediate(t *testing.T, kclient client.Client, ic *operatorv1.IngressController, annotationKey, expectedValue string) {
+	err := wait.PollImmediate(5*time.Second, 5*time.Minute, func() (bool, error) {
```

Comment:
We have started replacing the use of `wait.PollImmediate` with 
`wait.PollUntilContextTimeout(context.Background()`, as used in https://github.com/openshift/cluster-ingress-operator/blob/master/test/e2e/operator_test.go#L3189.  Please use the updated function here, and any new code that requires polled waiting.

Reply:
No problem and thanks for the suggestion. I have replaced `wait.PollImmediate` with `wait.PollUntilContextTimeout` to be consistent.